lib/solaar/cli/show.py

```diff
@@ -184,13 +183,15 @@ def _print_device(dev):
     if dev.online and dev.keys:
         print('     Has %d reprogrammable keys:' % len(dev.keys))
         for k in dev.keys:
-            flags = _special_keys.KEY_FLAG.flag_names(k.flags)
             # TODO: add here additional variants for other REPROG_CONTROLS
             if dev.keys.keyversion == 1:
-                print('        %2d: %-26s => %-27s   %s' % (k.index, k.key, k.task, ', '.join(flags)))
+                print('        %2d: %-26s => %-27s   %s' % (k.index, k.key, k.default_task, ', '.join(k.flags)))
             if dev.keys.keyversion == 4:
-                print('        %2d: %-26s, default: %-27s => %-26s' % (k.index, k.key, k.task, k.remapped))
-                print('             %s, pos:%d, group:%1d, gmask:%d' % (', '.join(flags), k.pos, k.group, k.group_mask))
+                print('        %2d: %-26s, default: %-27s => %-26s' % (k.index, k.key, k.default_task, k.mapped_to))
+                print(
+                    '             %s, pos:%d, group:%1d, group mask:%s' %
+                    (', '.join(k.flags), k.pos, k.group, ', '.join(k.group_mask))
```

Comment:
Nit:  I think that it would look better with no spaces between the groups in the group mask.